src/styles/components/header-bar/styles.less

```diff
@@ -9,8 +9,9 @@
     height: @header-bar-height;
     padding-left: @header-bar-horizontal-padding;
     padding-right: @header-bar-horizontal-padding;
+    text-rendering: optimizeLegibility;
     width: 100%;
-    z-index: 999;
```

Comment:
An additional "flash" was happening on the text because the dropdown panel would overlap the user's cursor, briefly turning off the hover style on the dropdown trigger